File: Code Chef/C Programming/200-299/203 Cricket World Cup Qualifier CWC23QUALIF.c
//Problem Simplification:
    //Check if team qualifies for next stage

//Logic:
    //x --> score of a team
    //if team scored 12 or more then it qualified & print "Yes"
    //otherwise print "No"
#include <stdio.h>
int main(void)
{
	int x;
	scanf("%d",&x);
	if(x>=12)
	{
	    printf("Yes\n");
	}
	else
	{
	    printf("No\n");
	}
	return 0;
}

File: Code Chef/JavaScript/200-299/203 Cricket World Cup Qualifier CWC23QUALIF.js
//Problem Simplification:
    //Check if any team qualifies to next stage

//Logic:
    //There'll be one input in each test case
    //Remove white space & convert it to integer
    //Call the function that contains the logic
    //If a team score 12 or more then it qualifies
    //Use ternary/simple if-else condition

process.stdin.setEncoding("utf-8");

const isQualified = (score) => {
    /* if(score > 11){
        console.log("YES");
    }else{
        console.log("NO");
    } */

    (score > 11)
        ? console.log("YES")
        : console.log("NO");
};

process.stdin.on("data", (input) => {
    const score = parseInt(input.trim());
    
    isQualified(score);
});